code/controllers/Processes/night_lighting.dm

```diff
@@ -60,26 +44,28 @@
 			if (isactive)
 				deactivate()
 
-/datum/controller/process/night_lighting/proc/activate()
-	for (var/obj/machinery/power/apc/APC in get_apc_list())
+// 'whitelisted' areas are areas that have nightmode explicitly enabled
+
+/datum/controller/process/night_lighting/proc/activate(var/whitelisted_only = 1)
+	isactive = 1
+	for (var/obj/machinery/power/apc/APC in get_apc_list(whitelisted_only))
 		APC.toggle_nightlight("on")
-		isactive = 1
 
 		SCHECK
 
-/datum/controller/process/night_lighting/proc/deactivate()
-	for (var/obj/machinery/power/apc/APC in get_apc_list())
+/datum/controller/process/night_lighting/proc/deactivate(var/whitelisted_only = 1)
+	isactive = 0
+	for (var/obj/machinery/power/apc/APC in get_apc_list(whitelisted_only))
 		APC.toggle_nightlight("off")
-		isactive = 0
 
 		SCHECK
 
-/datum/controller/process/night_lighting/proc/get_apc_list()
+/datum/controller/process/night_lighting/proc/get_apc_list(var/whitelisted_only = 1)
```

Comment:
We was thinking. Cutting the AI control wire (which also technically means remote control in general) should make the console also no longer affect said APCs. Consistency's sake, primarily.

```
if (B.apc && !B.apc.aidisabled)
	lighting_apcs += B.apc
```
Changing line 70 condition to that would cover it.